lib/Runtime/Library/SparseArraySegment.inl

```diff
@@ -352,7 +352,7 @@ namespace Js
         }
         dst->length = newLen;
         Assert(dst->length <= dst->size);
-        AssertMsg(srcIndex - src->left >= 0,"src->left > srcIndex resulting in negative indexing of src->elements");
+        AssertMsg(srcIndex >= src->left,"src->left > srcIndex resulting in negative indexing of src->elements");
```

Comment:
I have validated that this assert line of code is hit multiple times in the Unittests.